roles/pre_tasks/tasks/main.yml

```diff
@@ -1,10 +1,23 @@
 ---
 
 # Install absolutely necessary packages
+
+- name: Check if the absolutely necessary packages are installed
+  shell: "type {{ item }}"
```

Comment:
I'm also not very happy about using `type` because it relies on the fact that the package is named like the command it contains, that could change and makes the code confusing. I would rather suggest `rpm -q {{ item }}`.

Reply:
Thinking about it, you could even avoid a loop by doing something like `rpm -q {{ tower_pre_tasks_packages | join(' ') }}` and then in the next loop, change the condition to `when: (item + ' is not installed') in __towerutils_check_packages.stdout`.

e.g. the output of `rpm -q unzip untar 2>/dev/null` is:
```
unzip-6.0-49.fc33.x86_64
package untar is not installed
```

Reply:
Will do testing around this, Don't disagree with it, was more trying to find a way to verify that things were installed when you don't have root access.

Reply:
Got this working in most cases, but when testing here: https://github.com/sean-m-sullivan/tower_configuration/runs/1691976044

It skips the task somehow and then errors out because its not skipping the install absolutely necessary packages, Hoping to utilize this process for tower on both tower_utilties and tower_configuration builds so we can eval vs awx and tower.

Reply:
Updated, and tested, Looks like good changes!